packages/wonder-blocks-dropdown/components/dropdown.md

```diff
@@ -176,6 +176,7 @@ class MixedDropdownExample extends React.Component {
 
         return (
             <Dropdown
+                singleSelectOption
```

Comment:
I think we should use `singleSelectOption={true}` since it's less confusing.  Some people who are new to React may confuse `singleSelectOption` with `singleSelectOption={singleSelectOption}`.

Reply:
Sure, we also do something similar with `skipClientNav` which is why I did it here